webpack.dev.js

```diff
@@ -0,0 +1,37 @@
+const merge = require('webpack-merge')
+const common = require('./webpack.common.js')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = merge(common, {
+   devtool: 'inline-source-map',
+   devServer: {
+     contentBase: './public'
+   },
+
+   module: {
+    rules: [
+      {
+        test: /\s[a|c]ss$/,
+        exclude: /node_modules/,
+        loader: 'sasslint'
```

Comment:
I think I saw this loader in the `commons` file. You don't need to add it twice until you need to change a parameter or configuration.